components/brave_new_tab_ui/reducers/new_tab_reducer.ts

```diff
@@ -186,6 +187,14 @@ export const newTabReducer: Reducer<NewTab.State | undefined> = (state: NewTab.S
       break
     }
 
+    case Actions.dismissTogetherPrompt.getType(): {
```

Comment:
It looks like you are preferring not to add an exported constant to `new_tab_types`. Do you want to eventually remove the need for that file completely?

Reply:
Yes just trying to remove 1 layer of boilerplate. I've been using `redux-act` for Brave Today on the NTP via `today_actions.ts` and `reducers/today`, so just continuing that here for anything new.